src/Sylius/Behat/Context/Ui/Admin/ManagingCustomersContext.php

```diff
@@ -173,8 +173,38 @@ public function theCustomerWithNameShouldAppearInTheRegistry(CustomerInterface $
         $this->updatePage->open(['id' => $customer->getId()]);
 
         Assert::true(
-            $this->updatePage->isFullNameHasValue($firstName, $lastName),
+            $this->updatePage->isFullNameHasValues($firstName, $lastName),
             sprintf('Customer with name %s %s should exist but it does not.', $firstName, $lastName)
         );
     }
+
+    /**
+     * @Given I want to see all customers in store
+     */
+    public function iWantToSeeAllCustomersInStore()
+    {
+        $this->indexPage->open();
+    }
+
+    /**
+     * @Then /^I should see (\d+) customers in the list$/
+     */
+    public function iShouldSeeCustomersInTheList($amountOfCustomers)
+    {
+        Assert::true(
+            $this->indexPage->isAmountOfCustomersEqualTo((int)$amountOfCustomers),
```

Comment:
Yup, because ``isAmountOfCustomersEqualTo()`` count rows with customers, so you need integer to compare count of rows with expected amount. In other PR is a transformer for this casting (LexicalContext), which will be used here.